osbs/utils/yaml.py

```diff
@@ -0,0 +1,67 @@
+"""
+Copyright (c) 2020 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+
+
+from __future__ import print_function, absolute_import, unicode_literals
```

Comment:
We are not using the print function here (we were not using it in the other module either)

Reply:
Fixed.